stdlib/dynarray.ml

```diff
@@ -962,6 +962,42 @@ let for_all p a =
   check_same_length "for_all" a ~length;
   res
 
+let exists2 p a1 a2 =
+  let Pack {arr = arr1; length = length1; dummy = dummy1} = a1 in
+  let Pack {arr = arr2; length = length2; dummy = dummy2} = a2 in
+  check_valid_length length1 arr1;
+  check_valid_length length2 arr2;
+  if length1 <> length2 then invalid_arg "Dynarray.exists2";
```

Comment:
Same comment as for `Dynarray.for_all2`.

Reply:
Fixed.